lib/server/ServerRenderer.js

```diff
@@ -9,7 +9,14 @@ var ServerRenderer = {
     render: function(layout, view, environmentConfig, clientAppRequirePath, serverRequest) {
         var title = view.getTitle ? view.getTitle() : null;
         var metatags = view.getMetatags ? view.getMetatags() : null;
-        var appRoot = environmentConfig ? environmentConfig.appRoot : "";
+        var config = environmentConfig || {};
+        var appRoot = config.appRoot || "";
+        var clientAppBootstrapFunctionName = null;
+
+        if (typeof config.clientAppBootstrapFunctionName == "string") {
```

Comment:
Also comparing to a string literal may prevent V8 optimizing compiler from compiling this function.